app/uk/gov/hmrc/childcarecalculatorfrontend/views/result.scala.html

```diff
@@ -33,8 +33,9 @@
 }
 
 @changesToCircumstancesContent = {
```

Comment:
The name for this variable is not meaningful anymore, can we rename to something better. Is it needed?